### PR TITLE
tensorflow: fix build

### PR DESCRIPTION
**Pull request by DavidKorczynski**
This is not ready to be merged in

### CODE REVIEW

File: projects/tensorflow/BUILD [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# Fuzzing TensorFlow.
```
Comment (mihaimaruseac) [inline projects/tensorflow/BUILD:1]: Can't we use this from the TF repo? We'll continue updating the file there as more fuzzers get written and we get internal/external in sync.

If we need to have this here, then maybe we should remove the python ones, as we have a separate project for that.

**Review by mihaimaruseac — COMMENTED** [review]
Thank you for doing this. Just one comment for now to kick off discussion about this.

File: projects/tensorflow/BUILD (hunk 1 above)
Comment (DavidKorczynski) [reply projects/tensorflow/BUILD:1]: Yes! The reason I put this here was simply for testing whether the fix worked or not and keeping track. The real goal was to simply get the CI to pass and then move onwards with updating the Tensorflow repo.

I have been to busy on other things to finish this, but if you would like to resume the fuzzing of Tensorflow then I am happy to put in the efforts to get it working and migrate changes upstream.

**Comment by DavidKorczynski** [comment]
Closing this in favour of https://github.com/google/oss-fuzz/pull/6860

**Comment by mihaimaruseac** [comment]
Thank you so much for the fixes

**Pull request closed without merging**